Tests/SentryTests/Performance/SentryTracerTests.swift

```diff
@@ -395,6 +395,42 @@ class SentryTracerTests: XCTestCase {
         XCTAssertEqual(["key": 0], sut.data as! [String: Int])
     }
     
+    #if os(iOS)
```

Comment:
`l`:
```suggestion
    #if os(iOS) || os(tvOS)
```

Reply:
Profiling doesn't work on tvOS because the necessary APIs are not exported, same with watchOS. See SENTRY_TARGET_PROFILING_SUPPORTED. I did notice this should have macOS added to it, I will make that change.

Reply:
Changed this check to support macOS, iOS, and mac catalyst (but not tvOS or watchOS)